environment-py3.yml

```diff
@@ -0,0 +1,31 @@
+name: vmdirac-development
+
+channels:
+  - conda-forge
+
+dependencies:
+  # runtime
+  - python =3.8
+  - pip
+  - certifi
+  # testing and development
+  - autopep8
+  - caniusepython3
+  - coverage
+  - hypothesis
+  - ipython
+  - mock
+  - parameterized
+  - pylint >=1.6.5
+  - pytest >=3.6
+  - pytest-cov >=2.2.0
+  - pytest-mock
+  # code
+  - boto3
+  - apache-libcloud
```

Comment:
Looks like `boto3` is part of DIRACOS but not `apache-libcloud`. You can simply add it to https://github.com/DIRACGrid/DIRACOS/blob/master/config/requirements.txt